tests/test_lib.py

```diff
@@ -177,24 +177,22 @@ def test_print_and_return_output(self):
 class TestNodeLoad(ccmtest.Tester):
 
     def test_rejects_multiple_load_lines(self):
-        info = 'Load : 699 KB\nLoad : 35 GB'
+        info = 'Load : 699 KiB\nLoad : 35 GiB'
         with self.assertRaises(RuntimeError):
             ccmlib.node._get_load_from_info_output(info)
 
     def test_rejects_unexpected_units(self):
-        infos = ['Load : 200 PB', 'Load : 12 Parsecs']
+        infos = ['Load : 200 PiB', 'Load : 12 Parsecs']
 
         for info in infos:
             with self.assertRaises(RuntimeError):
                 ccmlib.node._get_load_from_info_output(info)
 
     def test_gets_correct_value(self):
-        info_value = [('Load : 328.45 KB', decimal.Decimal('328.45')),
-                      ('Load : 295.72 MB', decimal.Decimal('295.72') * 1024),
-                      ('Load : 183.79 GB',
```

Comment:
We still want correct handling of `GB` on versions that use it, so could you add tests for the `GiB` behavior, rather than replacing these? (and the same for other `X{,i}B` units)